.github/workflows/submitty_ci.yml

```diff
@@ -305,6 +305,15 @@ jobs:
           - 5432:5432
 
     steps:
+      - name: Delete huge unnecessary folders, and clean apt cache
+        run: |
+          rm -rf /opt/hostedtoolcache
+          rm -rf /usr/share/dotnet
+          rm -rf /opt/ghc
+          rm -rf "/usr/local/share/boost"
```

Comment:
Does this interfere with our usage of Boost?

Reply:
im not sure. It works without this line, so I have removed it.